traffic_portal/app/src/common/api/DeliveryServiceService.js

```diff
@@ -92,6 +92,17 @@ var DeliveryServiceService = function($http, locationUtils, messageModel, ENV) {
         )
     };
 
+    this.getSteering = function() {
+        return $http.get(`${ENV.api.unstable}steering/`).then(
+            function(result) {
+                return result.data.response;
+            },
+            function (err) {
+                throw err;
+            }
+        )
+    }
+
```

Comment:
because an error handler that does nothing but throw its argument is equivalent to default behavior, this statement can be written thusly on one line:

```javascript
this.getSteering = () => $http.get(`${ENV.api.unstable}steering/`).then(r=>r.data.response);
```
It doesn't need to be on one line or anything, but using arrow functions allows eliminating a lot of boilerplate, and the error "handler" can be omitted entirely.